uber/site_sections/preregistration.py

```diff
@@ -453,6 +453,9 @@ def transfer_badge(self, session, message='', **params):
     def confirm(self, session, message='', return_to='confirm', **params):
         attendee = session.attendee(params, bools=_checkboxes, restricted=True)
 
```

Comment:
Ooh, good point.

I'll try to remember when I re-implement all this stuff in master. :'D